src/test/groovy/pl/allegro/tech/build/axion/release/util/VersionComparatorTest.groovy

```diff
@@ -0,0 +1,90 @@
+package pl.allegro.tech.build.axion.release.util;
```

Comment:
remove along with `VersionComparator`. also there should be tests that test & describe new desired behaviour. since all old tests passed it means it did not break anything, but also there is nothing guarding this new logic, which is crucial for you, from being changed in the future